script/run-container.sh

```diff
@@ -32,6 +32,13 @@ TNF_INCLUDE_WEB_FILES_IN_OUTPUT_FOLDER_DEFAULT=false
 ON_DEMAND_DEBUG_PODS_DEFAULT=false
 TNF_ENABLE_DATA_COLLECTION_DEFAULT=false
 TNF_ENABLE_XML_CREATION_DEFAULT=false
+TIMEOUT_ARG=--timeout=${TIMEOUT:-24h0m0s}
```

Comment:
Ideally I believe that we should not need any scripts here. We should move support for offline db inside the CNFCERT container itself and just provide an example on how to run the CNFCERT container in the documentation using podman.

Reply:
I agree. I'm proceeding step by step here though. Trying first to decouple the scrips from one another, then prepare the binary to run in a standalone way efficiently and then deleting the scripts.